doc/changelog/07-vernac-commands-and-options/15652-depr-add-loadpath.rst

```diff
@@ -0,0 +1,4 @@
+- **Deprecated:**
+  :cmd:`Add LoadPath` and :cmd:`Add Rec LoadPath`
```

Comment:
```suggestion
  :cmd:`Add LoadPath` and :cmd:`Add Rec LoadPath` (use \"-Q\" or \"-R\" instead)
```

Reply:
That might be too ambiguous in the changelog however. Maybe also mention command-line/_CoqProject.